packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -129,11 +130,13 @@ export function normalizeError(
         fluidError.addTelemetryProperties({ typeofError: typeof (error) });
     }
 
-    const originalErrorTelemetryProps = isILoggingError(error) ? error.getTelemetryProperties() : {};
+    const originalErrorTelemetryProps = LoggingError.typeCheck(error)
```

Comment:
I was thinking about this after we talked last week - the alternative is to put this `LoggingError` check up on line 107. What I like about that approach is that it avoids creating a whole new error object for `LoggingErrors` with no `errorType`.

Reply:
Of course of course.  I realized that later in the day in the back of my mind :P  For now, I would keep this as-is, and open a follow-up item.  IMO the right thing here is:

A. `normalizeError` should return `LoggingError` not `IFluidErrorBase`
B. Some places that call `normalizeError` will need a new function similar to `DataProcessingError.wrapIfUnrecognized` which will tack on `errorType: genericError` if it's missing (anywhere that does something like `Container.close(normalizeError(error))`, for instance)

You can mention in that item that until this is fixed, calling `normalizeError` on a `LoggingError` will essentially clone the error object.  In practice I think this is fine (this PR is a good incremental step from the behavior today)